From 0e65bb2537b271f9a1b146d7e0a6d6630c4fb519 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Sun, 6 Dec 2020 20:47:11 -0500
Subject: fix def export

---
 goffice/Makefile.am | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/goffice/Makefile.am b/goffice/Makefile.am
index 71e2b2d4..8f966a9f 100644
--- a/goffice/Makefile.am
+++ b/goffice/Makefile.am
@@ -566,7 +566,16 @@ LIB_PUBLIC_HDRS = $(libgoffice_@GOFFICE_API_VER@_la_HEADERS)
 
 
 goffice.def: $(ALL_HDRS) $(top_srcdir)/tools/dumpdef.pl
-	cat $(top_builddir)/goffice/goffice-config.h $(ALL_HDRS) | \
+	hdrs='$(ALL_HDRS)'; \
+	hdrs_list=''; \
+	for hdr in $$hdrs; do \
+	  if test -f $(srcdir)/$$hdr; then \
+	    hdrs_list="$$hdrs_list $(srcdir)/$$hdr"; \
+	  else \
+	    hdrs_list="$$hdrs_list $$hdr"; \
+	  fi; \
+	done; \
+	cat $(top_builddir)/goffice/goffice-config.h $$hdrs_list | \
 	sed -e 's/^#[ 	]*include[ 	]\+.*$$//g' | \
 	$(CPP) $(AM_CPPFLAGS) "-DGO_VAR_DECL=__declspec(dllexport)" -P - > xgen-localdef.1
 	@PERL@ $(top_srcdir)/tools/dumpdef.pl \
-- 
