From 0abf578a0d12926620c93c57770f90b3139c7adc Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Thu, 17 Sep 2020 10:06:53 -0400
Subject: sdcc-extra: Disable gameboy

---
 sdcc-extra/emu/Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sdcc-extra/emu/Makefile.in b/sdcc-extra/emu/Makefile.in
index 1b6f3032a..0474359ab 100644
--- a/sdcc-extra/emu/Makefile.in
+++ b/sdcc-extra/emu/Makefile.in
@@ -3,6 +3,6 @@ VPATH = @srcdir@
 top_srcdir = @top_srcdir@
 top_builddir = @top_builddir@
 
-SUBDIRS = rrz80 rrgb
+SUBDIRS = rrz80
 
 include $(top_builddir)/Makefile.rules
-- 
2.28.0.windows.1

