From d22d052efef06bcd681d0ecc634c7a07e83d6028 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Sun, 30 Aug 2020 14:37:17 -0400
Subject: Add m6809 to port.h

---
 sdcc/src/port.h | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/sdcc/src/port.h b/sdcc/src/port.h
index ac3f595e8..8da5366ce 100644
--- a/sdcc/src/port.h
+++ b/sdcc/src/port.h
@@ -31,7 +31,8 @@
 #define TARGET_ID_PDK15    21
 #define TARGET_ID_PDK16    22
 #define TARGET_ID_Z80N     23
 #define TARGET_ID_R2KA     24
+#define TARGET_ID_M6809    25
 
 /* Macro to test the target we are compiling for.
    Can only be used after SDCCmain has defined the port
@@ -51,6 +53,7 @@
 #define TARGET_IS_Z80N     (port->id == TARGET_ID_Z80N)
 #define TARGET_IS_HC08     (port->id == TARGET_ID_HC08)
 #define TARGET_IS_S08      (port->id == TARGET_ID_S08)
+#define TARGET_IS_M6809    (port->id == TARGET_ID_M6809)
 #define TARGET_IS_STM8     (port->id == TARGET_ID_STM8)
 #define TARGET_IS_PDK13    (port->id == TARGET_ID_PDK13)
 #define TARGET_IS_PDK14    (port->id == TARGET_ID_PDK14)
@@ -63,9 +66,10 @@
 #define TARGET_HC08_LIKE   (TARGET_IS_HC08 || TARGET_IS_S08)
 #define TARGET_PIC_LIKE    (TARGET_IS_PIC14 || TARGET_IS_PIC16)
 #define TARGET_PDK_LIKE    (TARGET_IS_PDK13 || TARGET_IS_PDK14 || TARGET_IS_PDK15 || TARGET_IS_PDK16)
+#define TARGET_M6809_LIKE  (TARGET_IS_M6809)
 
 /* is using sdas / sdld assembler / linker */
-#define IS_SDASLD          (TARGET_Z80_LIKE || TARGET_MCS51_LIKE || TARGET_HC08_LIKE)
+#define IS_SDASLD          (TARGET_Z80_LIKE || TARGET_MCS51_LIKE || TARGET_HC08_LIKE || TARGET_M6809_LIKE)
 
 #define MAX_BUILTIN_ARGS        16
 /* definition of builtin functions */
@@ -482,5 +486,8 @@ extern PORT pdk14_port;
 #if !OPT_DISABLE_PDK15
 extern PORT pdk15_port;
 #endif
+#if !OPT_DISABLE_M6809
+extern PORT m6809_port;
+#endif
 
 #endif /* PORT_INCLUDE */
-- 
