From 2ad35e875a938bb24e7eda58989173d20a3da52e Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Sat, 5 Dec 2020 20:03:28 -0500
Subject: msys allow-multiple-definition

---
 src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index f7102082d..92cfbdcf1 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -671,7 +671,7 @@ if WITH_NATIVE_WIN32
     ssgrep_LDFLAGS	+= -mconsole
 
     libspreadsheet_la_DEPENDENCIES = libspreadsheet.def
-    libspreadsheet_la_LDFLAGS += -no-undefined -export-symbols libspreadsheet.def
+    libspreadsheet_la_LDFLAGS += -Wl,--allow-multiple-definition -no-undefined -export-symbols libspreadsheet.def
 endif
 
 if WITH_WIN32
-- 
2.29.2.windows.2

