From 8f53e8f01419121d3fba16dd0196f5d0bc399bbb Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Sat, 19 Dec 2020 01:20:53 -0500
Subject: allow libtool to link to static libs

---
 configure.ac | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 067471f86..94643d03f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -98,8 +98,7 @@ AC_PROG_LN_S
 AC_HEADER_STDC
 
 LT_PREREQ([2.2.6])
-# Make --disable-static the default
-LT_INIT([disable-static])
+LT_INIT()
 
 dnl Propagate Gnome-specific variable ACLOCAL_FLAGS to Makefile.
 AC_SUBST(ACLOCAL_AMFLAGS, $ACLOCAL_FLAGS)
-- 
